react/src/pages/participants/SampleTable.tsx

```diff
@@ -1,54 +1,118 @@
-                        field: "created",
-                        render: rowData => formatDateString(rowData.created),
-                    },
-                    { title: "Create By", field: "created_by" },
-                    {
-                        title: "Update Time",
-                        field: "updated",
-                        render: rowData => formatDateString(rowData.updated),
-                    },
-                    { title: "Updated By", field: "updated_by" },
-                ]}
-                data={samples}
-                title="Samples"
-                detailPanel={rowData => <DatasetAccordions datasets={rowData.datasets} />}
-                options={{
-                    paging: false,
-                    selection: false,
-                    search: false,
-                    padding: "dense",
-                }}
-            />
-        </div>
+        <MaterialTable
+            columns={[
+                { title: "Sample ID", field: "tissue_sample_id", cellStyle: cellStyle },
+                {
+                    title: "Extraction Date",
+                    field: "extraction_date",
+                    cellStyle: cellStyle,
+                    render: rowData => formatDateString(rowData.updated),
+                },
+                { title: "Sample Type", field: "tissue_sample_type", cellStyle: cellStyle },
+                { title: "Tissue Processing", field: "tissue_processing", cellStyle: cellStyle },
+                { title: "Notes", field: "notes", cellStyle: cellStyle },
+                {
+                    title: "Creation Time",
+                    field: "created",
+                    cellStyle: cellStyle,
+                    render: rowData => formatDateString(rowData.created),
+                },
+                { title: "Create By", field: "created_by", cellStyle: cellStyle },
+                {
+                    title: "Update Time",
+                    field: "updated",
+                    cellStyle: cellStyle,
+                    render: rowData => formatDateString(rowData.updated),
+                },
+                { title: "Updated By", field: "updated_by", cellStyle: cellStyle },
+            ]}
+            data={samples}
+            title="Samples"
+            detailPanel={rowData => {
+                const infoList: Info[] = rowData.datasets.map(dataset => {
+                    return {
+                        primaryListTitle: `Dataset ID ${dataset.dataset_id}`,
+                        titles: getTitles(),
+                        values: getValues(dataset),
+                    } as Info;
+                });
+                return (
+                    <div className={classes.datasetList}>
+                        <InfoList infoList={infoList} icon={<Dns />} />
+                    </div>
+                );
+            }}
+            components={{
+                Container: props => <div>{props.children}</div>,
```

Comment:
Since it looks like you're trying to remove padding for all cells in the SampleTable, you can do it in one place by overriding the Cell component like so:
```suggestion
                Container: props => <div>{props.children}</div>,
                Cell: props => <MTableCell {...props} style={cellStyle} />
```